### DISCUSSION TITLE
Improve Compiler Speed Documentation

### DISCUSSION THREAD

**Discussion by chinmaykh** [themesberg/flowbite-svelte#838] [Ideas]
The [requirements section](https://flowbite-svelte.com/docs/pages/compiler-speed) is extremely important. The note on avoiding complaints by the editor could be under another subheading / in the beginning. The current docs makes it looks as if the requirements section is applicable only for TS users which it is not. Additionally, the text `tsconfig.json` file could be reworded to `.svelte-kit/tsconfig.json` as most JS+JSDoc users do not know where it is.

**Comment by shinokada** [comment]
Can you create a PR?
